roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,24 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
+
+- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
+- Act as a conduit of information between the Rust Project and the Rust Foundation
+
+## Time Budget
+
+- 2-5 hours per week
```

Comment:
Is this realistic? I'm not sure I would say that over a year I see time averaging out to 5 hours/week, even though some weeks are that high (and on rare occasions higher).

Reply:
Yes, I believe so. It's based on the surveying Paul and Abi had to do back in sept of last year for our 990 filing. The email was titled "Average time spent by Board members working on the Rust Foundation". The chair reported spending 5 hours per week, other members averaged around 2.5 hours per week. This happened to be right around the time I was heavily using pomodoros for time tracking so I actually had accurate timecard data of how much time I spent on foundation work (minus zulip time) which came out to 19 hours over 3 months or approximately 1.5 hours per week.

Reply:
Hm, okay. I wonder how much bias there is in terms of not doing "too little work" when asked how much work you're doing :)


But I think that helps explain the 5 hours and the rest falls down naturally from there, so that seems okay to me.